### DISCUSSION TITLE
Missing User Fields in Response when requesting List Members

### DISCUSSION THREAD

**Discussion by skriptum** [tweepy/tweepy#1744] [Questions] [answered]
Hi, 
I try to get the Members of a List and some Information via the `Client.get_list_members()` function.
When I define parameters for the `user_field` and make the call to the API, the Response does not include the requested information

## Steps to reproduce
an example call to the api
```python
cli = tweepy.Client(bearer_token="TWITTER_BEARER_TOKEN")

r = cli.get_list_members(
    973614171886686209, 
    max_results=1, 
    user_fields= ["description","created_at"] #changing this to string doesnt change anything either
    )

print(r.data)
```
Results in :
```
[<User id=1416388709264633858 name=Henning Otte username=vtdgsprecher>]
```

## Expected Behaviour
i try to reproduce everything with the `requests`package
```python
url = "https://api.twitter.com/2/lists/973614171886686209/members"
params = {
    "user.fields":"created_at",
    "max_results":1
    }
headers= {"Authorization": f"TWITTER_BEARER_TOKEN"}

r = requests.get(url = url, params = params, headers=headers)
r.json()["data"]
```
This produces the Output I want: (altough in other format, but not important here)
```python
[{'id': '1416388709264633858',
  'description': 'Mitglied des Verteidigungsausschusses, Bundesmitgliederbeauftragter der @CDU', #the wished for fields
  'created_at': '2021-07-17T13:26:19.000Z', 
  'username': 'vtdgsprecher',
  'name': 'Henning Otte'}]
```
as you can see, this time the requested field is included in the data field for the user

## Steps to Fix?

honestly I dont know whats going wrong here, I tested a bunch of things and looked at the [code here](https://github.com/tweepy/tweepy/blob/e3bfad54e2914d38fdf8b8e10c2bedd7539f9363/tweepy/client.py#L2226) but i still dont see a solution.

Maybe im just missing something super obvious here

Thanks for your help

**Comment by Harmon758** [comment, marked as answer]
The string representation of API v2 models, including `User`, only includes the default fields of the particular model.
You can access the other fields as attributes of the model, e.g. `r.data[0].description`.
See the [relevant FAQ section in the documentation](https://tweepy.readthedocs.io/en/v4.4.0/faq.html#why-am-i-not-getting-expansions-or-includes-data-with-api-v2-using-client) for more information.
